### PR TITLE
[WIP]feat: New design for array ui

### PR DESCRIPTION
**Pull request by alanlong9278**
## Description

 New design for array ui.
### Done
- Set input placeholder font-style `Italic`
- Set `MaxTurnCount`'s minValue 1. 
- Add icon for `expression rules` input.

### TODO

- [ ] Choices UI

- [ ] Prompt setting

## Task Item

#1510 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code refactor (non-breaking change which improve code quality, clean up, add tests, etc)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Doc update (document update)

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have functionally tested my change

## Screenshots

Please include screenshots or gifs if your PR include UX changes.

### CODE REVIEW

File: Composer/packages/extensions/obiformeditor/src/Form/fields/styles.css [hunk 1]
```diff
@@ -54,3 +54,7 @@
 a {
   color: black;
 }
+
+input::placeholder {
+  font-style: italic;
+}
```
Comment (a-b-r-o-w-n) [inline Composer/packages/extensions/obiformeditor/src/Form/fields/styles.css]: Let's not make this a global style. Also, moving forward, only use css-in-js

File: Composer/packages/extensions/obiformeditor/src/Form/fields/PromptField/index.tsx [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const PromptField: React.FC<BFDFieldProps> = props => {
           <PivotItem headerText={formatMessage('Bot Asks')} itemKey={PromptTab.BOT_ASKS}>
             <BotAsks {...props} onChange={updateField} getSchema={getSchema} />
           </PivotItem>
-          <PivotItem headerText={formatMessage('User Input')} itemKey={PromptTab.USER_INPUT}>
```
[the remark this answers is not in the document]
Comment (alanlong9278) [reply Composer/packages/extensions/obiformeditor/src/Form/fields/PromptField/index.tsx]: revert it

File: Composer/packages/extensions/obiformeditor/src/Form/fields/styles.css (hunk 1 above)
Comment (alanlong9278) [reply Composer/packages/extensions/obiformeditor/src/Form/fields/styles.css]: image
It should be a global style according to @sangwoohaan

**Comment by cwhitten** [comment]
hey @alanlong9278 - thanks for getting this going, but @tdurnford is actively working on the array ui changes.

**Comment by a-b-r-o-w-n** [comment]
sorry for the mix-up @alanlong9278. Closing this for now.

**Pull request closed without merging**